src/ishikawa_backend.erl

```diff
@@ -47,7 +47,11 @@
 
 -record(state, {actor :: actor(),
                 vv :: timestamp(),
-                members :: [term()]}).
+                members :: [term()],
```

Comment:
`node()` is a built in, and it's what should be returned from the partisan membership library.